modules/api/README.md

```diff
@@ -0,0 +1,38 @@
+# API
+
+## Purpose
+
+The api module is intended to provide a stable, versioned REST interface
+for interacting with the core LORIS functionality without requiring backend
+SQL access.
+
+## Intended Users
+
+The module has 2 intended users:
+1. Users (or creators) of LORIS data that are external to the LORIS core
+   software that must extract or import data from LORIS.
+2. Internal LORIS modules
+
+## Scope
+
+The API only provides information about concepts that are cross-module
+or data that is of general concern to users. 
+
+NOT in scope:
+
+- Data and functionality that is specific to a single module, and not of general
+  nature.
+- A graphical frontend
+
+## Permissions
+
+????
+
+## Configurations
+
+useEDC - This configuration setting determines whether the EDC is returned
+  or required for creating/retrieving candidates with the API.
+
+## Interactions with LORIS
+
+???
```

Comment:
Same here

Reply:
@driusan we could list the current API interactions with LORIS.
Such as candidates, login, projects, and update this when new API gets added. thoughts? @xlecours

Reply:
Yes, that's what I was thinking.. I just don't know the current extent of which modules affect where the data is coming from.

Reply:
i think modules are not "users" but should go here, as a general mention.

Reply:
The API module is different than other modules in that it's intended to be an API, as such I think modules are (or, at least, should be) users of the API.